### DISCUSSION TITLE
Unresponded network requests might result in timeout errors in Detox tests

### DISCUSSION THREAD

**Discussion by VasekRi** [wix/Detox#3870] [Q&A]
Hello,

I have an issue with detox test when testing React Native app. Tests are failing on 120000ms timeout. But test build is successful, app is running in simulator without any issues. The main screen of the app is accessible and fully responsive. So the first element in the test scenario is visible and should be executable.

I am using Detox also with cucumber and Gherkin. But I also tried it only with Detox CLI, but having the same issue. 

Every time the test are hanging on the queue.
![image](https://user-images.githubusercontent.com/104826198/214504338-d6e6bdb6-db03-41bd-b066-bf9068214fd2.png)

And when timeout runs out, there is FAIL with:

 The app has not responded to the network requests below:
  (id = -1000) isReady: {}
Unresponded network requests might result in timeout errors in Detox tests.

Options tried:
https://stackoverflow.com/questions/55820649/react-native-detox-cant-accept-location-permission-from-system-alert-pop-up
https://stackoverflow.com/questions/64932555/setting-app-permissions-for-detox-ios-tests-not-working-properlyhttps://github.com/wix/Detox/pull/2009#issuecomment-648971528
https://www.tabnine.com/code/javascript/modules/detox
https://stackoverflow.com/questions/72044950/detox-tests-hang-with-pending-items-on-dispatch-queue

Shall I ask you for some tips how to fix that?

**Comment by Seynab150** [comment]
Hello, did you find a solution as I am running into the same exact problem. Thank you!

**Comment by VasekRi** [comment]
Hello,
 
unfortunately I did not. It seemed that in that company they used some kind of React Native library. So then it was web view which was probably causing the issue. 
I did not find the solution from detox side. So they had to use some different approach on Frontend (I do not know what they changed) and after that the issue was not there. I am not working there anymore so I could not give you the answer what was the root of this problem, sorry.
 
Best regards.
______________________________________________________________
> Od: "Seynab150" ***@***.***>
> Komu: "wix/Detox" ***@***.***>
> Datum: 30.12.2024 11:10
> Předmět: Re: [wix/Detox] Unresponded network requests might result in timeout errors in Detox tests (Discussion #3870)
>
 
Hello, did you find a solution as I am running into the same exact problem. Thank you!
